src/components/internals/clear-indicator/clear-indicator.js

```diff
@@ -1,28 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import { useIntl } from 'react-intl';
 import { CloseIcon } from '@commercetools-uikit/icons';
+import messages from './messages';
 
 const ClearIndicator = props => {
+  const intl = useIntl();
   const {
     getStyles,
     innerProps: { ref, ...restInnerProps },
   } = props;
   return (
-    <div
+    <button
       {...restInnerProps}
       ref={ref}
+      css={css`
+        border: none;
+        cursor: pointer;
+        background: none;
+        box-sizing: border-box;
+        text-decoration: none;
+      `}
       style={getStyles('clearIndicator', props)}
+      title={intl.formatMessage(messages.clearButtonLabel)}
+      aria-label={intl.formatMessage(messages.clearButtonLabel)}
     >
-      <CloseIcon color={props.isDisabled && 'neutral60'} size="medium" />
```

Comment:
Removed this because it is never disabled. When the Input is disabled/readonly, this component isn't rendered.